alarm/management/commands/get_binance_price_and_analized.py

```diff
@@ -0,0 +1,14 @@
+from django.core.management.base import BaseCommand, CommandError
```

Comment:
I'd like to name file something like "price_break_threashold_alarm" or something like this.

Reply:
++